examples/example-gateway/endpoints/googlenow/endpoint-config.json

```diff
@@ -4,8 +4,8 @@
 	"config": {
 		"rateLimit": 100,
 		"endpoints": [
-			"./add_credentials.json",
-			"./check_credentials.json"
+			"add_credentials.json",
```

Comment:
My current delete endpoint implementation expects the entry inside the array to be [handleId in snake case].json, which is exactly the convention used when an endpoint is added in the code. It seems that some of the examples don't follow this convention. To support a prefix like `./` , I can do an absolute path comparison, which would eliminate something like `./`. To support a filename that isn't [handleId in snake case], I'll have to basically try to read every file listed in the array and check its contents to see which endpoint it is. What do you think is the best way?

Reply:
which example doesn't match the snake case matching? I think we need to follow the same naming convention. Either way works as long as it's consistent

Reply:
There could be more, but bar.helloWorld has handleId helloWorld, but the json is named hello.json. I agree we need to follow the same convention. Is the convention defined as how the code adds a new endpoint? If so, then the examples aren't following it consistently. Would you prefer I adjust the examples to follow them, or change my delete implementation so it also supports the current examples?

Reply:
I guess for best compatibility I'll make it so that if there's no match based on the absolute path, it'll try to read each json to see if the endpoint is actually there